src/components/Accounts/Actions/Actions.js

```diff
@@ -469,7 +469,10 @@ class Actions extends React.Component {
     const waives = _.get(resources, ['waives', 'records'], []);
     const transfers = _.get(resources, ['transfers', 'records'], []);
     const settings = _.get(resources, ['commentRequired', 'records', 0], {});
-    const warning = accounts.filter(a => a.status.name === 'Closed').length !== 0 && (actions.regular || actions.waiveMany || actions.transferMany) && params.accountstatus;
+    const closedAccounts = accounts.filter(a => a.status && a.status.name === 'Closed');
+    const isWarning = closedAccounts.length !== 0
+        && (actions.regular || actions.waiveMany || actions.transferMany)
```

Comment:
What do you think about
``const hasClosedAccounts = accounts.some(a => a.status && a.status.name === 'Closed');``
``const isWarning = hasClosedAccounts``
``...`` ?

Reply:
It could make code more readable, but I guess here should not be used ternary operator, because value of `isWarning` boolean is used in several places inside the conditions